lib/generators/deimos/active_record_consumer/templates/migration.rb.tt

```diff
@@ -0,0 +1,30 @@
+class <%= migration_class_name %> < ActiveRecord::Migration<%= migration_version %>
```

Comment:
Don't we already have this template file? We should be able to reuse it.

Reply:
Yes - better idea to reuse the existing code - will fix this.